cmd/skaffold/app/cmd/completion.go

```diff
@@ -245,11 +74,8 @@ func NewCmdCompletion() *cobra.Command {
 }
 
 func runCompletionZsh(cmd *cobra.Command, out io.Writer) {
-	io.WriteString(out, zshInitialization)
-	buf := new(bytes.Buffer)
-	rootCmd(cmd).GenBashCompletion(buf)
-	out.Write(buf.Bytes())
-	io.WriteString(out, zshTail)
+	_ = rootCmd(cmd).GenZshCompletion(out)
+	_, _ = io.WriteString(out, zshCompdef)
```

Comment:
Could it be?
```
rootCmd(cmd).GenZshCompletion(out)
io.WriteString(out, zshCompdef)
```

Reply:
Sure. Only my IDE complains about it, so I explicitly ignored the errors.